packages/core/src/components/graphs/gauge.ts

```diff
@@ -87,7 +87,7 @@ export class Gauge extends Component {
 		}
 	}
 
-	render(animate = true) {
+	render(animate: boolean = true) {
```

Comment:
the fact that `animate` is a boolean is inferred **automatically** by typescript from the fact that you're setting it to `true`